contributors/design-proposals/storage/volume-topology-scheduling.md

```diff
@@ -194,12 +245,11 @@ and region labels.  This will handle newly created PV objects.
 
 Existing PV objects will have to be upgraded to use the new NodeAffinity field.
 This does not have to occur instantaneously, and can be updated within the
-deprecation period.
+deprecation period.  This can be done through a new PV update controller that
```

Comment:
Do we need to explicitly handle downgrade? Regardless let's call out both upgrade and downgrade explicitly in the doc.

Reply:
Moved Zonal PV Upgrade section up and also added some details about downgrade.

Also after some thought and more discussion with @saad-ali and @davidz627, we won't support automatic upgrade of StorageClasses to use AllowedTopologies.  Admins have to create a new StorageClass anyway to use the delayed binding, so at that time, we can encourage users to use AllowedTopologies instead of zone parameters.